### PR TITLE
BIC-188 # refactor Pending queue

### PR DESCRIPTION
**Pull request by ashishtilara**
- model-application: Extendable pending-queue view placeholder
- moved pending queue related moustache templates in template folder
- view/form/pending: split pending queue view specific functions from view-interaction to separate view class
- added tests in view-interaction that confirms preparePendingQueueSubView() functions correctly
-- and picks correct view
- md file that shows what variables are available for pending queue default view

### CODE REVIEW

File: src/bic/view/form/pending.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+  // this module
+
+  PendingQueue = Backbone.View.extend({
+
+    events: {
+      'click .clearPendingItem': 'clearPendingItem',
+      'click #submitPendingItems': 'submitPendingItems',
+      'click #clearPendingItems': 'clearPendingItems',
+      'click #clearPendingItemsConfirmation': 'clearPendingItemsConfirmation'
+    },
```
Comment (simonmarklar) [inline src/bic/view/form/pending.js]: we changed the structure of event selector name to `[data-onevent="functionbToCall"]` eg `[data-onclick="clearPendingItem"]`, might as well update this while refactoring :)

Comment (ashishtilara) [inline src/bic/view/form/pending.js]: Thats change of functionality, might do it in separate pull request, once this one is merged.

**Comment by jokeyrhyme** [comment]
LGTM :+1:
